lib/beamchmark/math.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Beamchmark.Math do
-  @moduledoc false
-  # module containing math utility functions
+  @moduledoc """
+  The module contains helper math types and utility functions.
```

Comment:
Not sure if this should be a public module

Reply:
I have a problem with this one, because `SchedulerInfo` refers to types defined in `Math` module. So we can either expose `Math` or copy-paste type definitions from there I guess?

Reply:
Ok let's leave it public